.github/workflows/release.yml

```diff
@@ -32,7 +41,49 @@ jobs:
 
       - name: Publish Events CLI image
         run: |
-          docker build . -t $EVENTS_CLI_LATEST_TAG -t $EVENTS_CLI_TAG -t $EVENTS_CLI_VERSION_TAG && \
+          docker build . -t $EVENTS_CLI_LATEST_TAG \
+          -t $EVENTS_CLI_TAG \
+          -t $EVENTS_CLI_VERSION_TAG \
+          --label "faros-build-id=$GITHUB_RUN_ID" \
+          --label "faros-ci-pipeline=$FAROS_PIPELINE_ID" \
+          --label "faros-ci-org=faros-ai" \
+          --label "faros-ci-source=GitHub" \
+          --label "faros-commit-sha=$GITHUB_SHA" \
+          --label "faros-vcs-repo=faros-events-cli" \
+          --label "faros-vcs-org=faros-ai" \
+          --label "faros-vcs-source=GitHub" && \
           docker push $EVENTS_CLI_LATEST_TAG && \
           docker push $EVENTS_CLI_TAG && \
           docker push $EVENTS_CLI_VERSION_TAG
+
+      - name: Set CI end time
+        id: set-ci-end-time
+        run: echo "CI_ENDED_AT=$(date +%s%N | cut -b1-13)" >> $GITHUB_ENV
+      
+      - name: Send CI event to Faros - prod
+        uses: faros-ai/faros-cicd-github-action@v2.0.0
```

Comment:
Actually, since i have access to the faros_event.sh script, what do you think about just calling it directly rather than using the github action?

Reply:
Yeah, good idea.

Reply:
Whatever is easier